src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -730,3 +731,27 @@ export function updateBreadcrumb(breadcrumb) {
     dispatch({ type: UPDATE_BREADCRUMB, breadcrumb });
   };
 }
+/**
+ * Function to retrieve provider information from the provider
+ * endpoint when using the v2 api.
+ *
+ * @param {*} appointments
+ */
+export function getProviderInfoV2(appointments) {
+  // Provider information included with v2 provider api call.
+  return async (dispatch, getState) => {
+    const featureVAOSServiceCCAppointments = selectFeatureVAOSServiceCCAppointments(
+      getState(),
+    );
+    if (featureVAOSServiceCCAppointments) {
+      const ProviderNpi = getPreferredCCProviderNPI(appointments);
+
+      const providerData = await getTransformedPreferredProvider(ProviderNpi);
```

Comment:
I would remove 'Transformed' from the function name or better yet, change 'getTransformedPreferredProvider' to 'fetchPreferredProvider' to follow naming standard of functions in services/index.js. It's not 100% but the original intent was for the 'get' functions to be helper functions for retrieving data.